compiler/src/main/scala/org/scalajs/core/compiler/GenJSCode.scala

```diff
@@ -5534,6 +5551,12 @@ abstract class GenJSCode extends plugins.PluginComponent
     settings.Xexperimental.value
   }
 
+  // The new collections have been introduced in 2.13.0-M4
+  private[scalajs] lazy val isScala213NewCollections = {
+    val v = scala.util.Properties.versionNumberString
+    v.startsWith("2.13") && v != "2.13.0-M3"
+  }
```

Comment:
We check that we are on a 2.13 release that contains the new collections. I’m not sure if I should exclude the previous milestones as well since they are not mentioned either in the build.